src/js/components/TaskMesosUrlComponent.jsx

```diff
@@ -0,0 +1,65 @@
+var React = require("react/addons");
+
+var InfoActions = require("../actions/InfoActions");
+var InfoEvents = require("../events/InfoEvents");
+var InfoStore = require("../stores/InfoStore");
+
+var TaskMesosUrlComponent = React.createClass({
+  displayName: "TaskMesosUrlComponent",
+  propTypes: {
+    className: React.PropTypes.string,
+    task: React.PropTypes.object.isRequired,
+    text: React.PropTypes.string
+  },
+
+ getInitialState: function () {
+    return {
+      info: InfoStore.info
+    };
+  },
+
+  componentDidMount: function () {
+    InfoActions.requestInfo();
+  },
+
+  componentWillMount: function () {
+    InfoStore.on(InfoEvents.CHANGE, this.onInfoChange);
+  },
+
+  onInfoChange: function () {
+    this.setState({
+      info: InfoStore.info
+    });
+  },
+
+  buildUrl: function (task, info) {
```

Comment:
I do not agree with that. You can easily still add the task object as an argument to the function to build the url. Like I said if not I think it is fine to have within the `TaskListItemComponent`, I don;t see the need for a whole component for this very specific purpose, unless, of course, we create a component for links in general, but then I still don't see this function belonging inside of the component, but being build by the user of the component.